examples/hacktoberfest_example.py

```diff
@@ -51,7 +56,10 @@ def get_prs(repo):
     prs = get_prs(repo)
     for pr in prs:
         text = "{} by {} is worth {} points".format(pr['url'], pr['author'], pr['points'])
-        num_reviewers = len(pr['reviewers'])
+        try:
+            num_reviewers = len(pr['reviewers'])
+        except Exception:
+            num_reviewers = 0
```

Comment:
Prefer ` num_reviewers = len(pr['reviewers'] or [])`. Exceptions should be reserved for exceptional cases.